scripts/is-it-pretty.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+node_modules/.bin/prettier-standard '**/*.js'
```

Comment:
Instead of running `prettier-standard` and checking for git changes we could run `standard` and check for a non-empty response. If the response is non-empty it means there’s an error and we can print the error so the failed build has a helpful response.

Reply:
This is what gets printed now: https://travis-ci.org/ViacomInc/data-point/jobs/335283922#L782

Reply:
standard can pass in many cases where prettier wont, I think we need either a new task to be standard only, or as a follow up add it to this task, for now I think this PR is good to only run prettier